src/main/java/org/apache/commons/text/names/HumanNameParser.java

```diff
@@ -131,20 +135,24 @@ public Name parse(String name) {
 
         NameString nameString = new NameString(name);
         // TODO compile regexes only once when the parser is created
+        String salutations = StringUtils.join(this.salutations, " |") + "";
```

Comment:
Should `""`  be `" "` so that `dr` is treated the same way as the other salutations? e.g. the other salutations will have a ` ` appended to them, `dr` will not.